source/draw/gpu/opengl/backend/gles3/Gles3ShaderProgram.ooc

```diff
@@ -86,6 +87,31 @@ Gles3ShaderProgram: class extends GLShaderProgram {
 		glUniform4f(glGetUniformLocation(this _backend, name), value x, value y, value z, value w)
 		version(debugGL) { validateEnd("ShaderProgram setUniform~FloatPoint4D") }
 	}
+	setUniform: override func ~ColorBgr (name: String, value: ColorBgr) {
+		version(debugGL) { validateStart("ShaderProgram setUniform~ColorBgr") }
+		glUniform3f(glGetUniformLocation(this _backend, name), (value blue as Float) / 255.0f, (value green as Float) / 255.0f, (value red as Float) / 255.0f)
```

Comment:
Can't you implement a `normalized` property returning `Float/FloatPoint2D/FloatPoint3D/FloatPoint4D` for every `Color` instead? I've already implemented it for `ColorBgra`. Then all these could be simplified to:
```ooc
this setUniform(name, value normalized)
```
OR we keep this layer primitive and only allow Int and Float Size/Point/Array, which means the conversion needs to be done when binding the shader variable:
```ooc
map add("color", color normalized)
```

What do you think?